src/cloud-cli/applications/update-app.ts

```diff
@@ -2,11 +2,17 @@ import axios from "axios";
 import { GlobalLogger } from "../../telemetry/logs";
 import { getCloudCredentials } from "../utils";
 import { Application } from "./types";
+import path from "node:path";
 
-export async function updateApp(appName: string, host: string, machines: number): Promise<number> {
+export async function updateApp(host: string, machines: number): Promise<number> {
```

Comment:
I realize we now do have a very tight coupling between package.json's application name and our application record in the database. When users update their app name, they have to update the application name too. (Which our API does not support today.)